docs/fundamentals/full-stack.md

```diff
@@ -76,7 +76,7 @@ export class AppComponent {
 }
 ```
 
-Next, let's create the api. We can do it by running `ng g node-application api --frontend-project=frontend` (`--frontend-project=frontend` set ups the proxy configuration such that the frontend application can access the api).
+Next, create the api. You can do it by running `ng g node-application api --frontend-project=frontend` (`--frontend-project=frontend` set ups the proxy configuration such that the frontend application can access the api).
```

Comment:
We need to implement this.

I created an issue: https://github.com/nrwl/nx/issues/1041